ide/coq_lex.mll

```diff
@@ -23,7 +23,11 @@ let number = [ '0'-'9' ]+
 
 let string = "\"" _+ "\""
 
-let undotted_sep = (number space* ':' space*)? '{' | '}' | '-'+ | '+'+ | '*'+
+let alpha = ['a'-'z' 'A'-'Z']
+
+let ident = alpha (alpha | number | '_' | "'")*
```

Comment:
For now, this is a clear under-approximation. I'm not against some help in making it less so.